src/styles/_catalog.scss

```diff
@@ -1,22 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
 
+  gap: 48px;
   max-width: 944px;
   margin: 0 auto;
+  justify-content: center;
+
+  @media (max-width: 488px) {
+    grid-template-columns: repeat(1, 200px);
+  }
+
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (max-width: 1024px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1025px) {
+    grid-template-columns: repeat(4, 200px);
+  }
```

Comment:
this is redundant, place this property under the 'catalog' class

Reply:
also other media queries isn't working because of that
<img width="829" alt="Знімок екрана 2022-10-31 о 10 46 46" src="https://user-images.githubusercontent.com/106310796/198968314-dd225568-34fd-444a-88d1-a693fade7afe.png">
<img width="556" alt="Знімок екрана 2022-10-31 о 10 47 28" src="https://user-images.githubusercontent.com/106310796/198968317-16d3dcd4-4446-4f8d-872b-a8f93e131004.png">